README.md

```diff
@@ -23,10 +23,15 @@ This style guide is a compilation of [atomic components](http://bradfrost.com/bl
 
 - `cd styleguide`
 - `composer install`
-- `npm install`
+- `sudo npm install -g npm`
 - `sudo npm install -g gulp`
   - This will install gulp globally on your machine. If you don't have privileges, don't want to install globally, or need to manage multiple projects using `gulp`, you can invoke `gulp` via `./node_modules/.bin/gulp serve` instead of directly.
 
+**Version Requirements**
+
+**Node.js** - v7.10.1 ( always < v8.x as this has breaking conflicts)
+**npm** - >= v5.2
```

Comment:
Update these to bullets, they show as one line and it's easy to miss the npm requirement when skimming (or only passively reading). Also you should add node to update npm to the mac setup IMO.